### PR TITLE
executor: fix goword checker bugs in insert*.go

### PR DESCRIPTION
**Pull request by zoomxi**
### What problem does this PR solve?

Problem Summary:  according to the discuss in PR #26189  , i make a PR to fix goword checker bugs in insert*.go first, and after this i will make another PR for issue fix.

### What is changed and how it works?

What's Changed: executor/insert*.go

How it Works: fix goword checker bugs

### Check List 

Tests 

- [ ] Unit test

### Release note 

- No release note

### CODE REVIEW

File: executor/insert_test.go [hunk 1]
```diff
@@ -1571,7 +1571,7 @@ func combination(items []string) func() []string {
 	}
 }
 
-// See https://github.com/pingcap/tidb/issues/24582
+// TestDuplicatedEntryErr : See https://github.com/pingcap/tidb/issues/24582
```
Comment (mmyj) [inline executor/insert_test.go:1574]: Delete `:`.

Comment (zoomxi) [reply executor/insert_test.go]: ok, thanks for reply

Comment (mmyj) [reply executor/insert_test.go]: You should delete other `:` also. 😆

Comment (zoomxi) [reply executor/insert_test.go]: haha, okay

**Comment by mmyj** [comment]
Following is the all `goword` errors of the upstream master branch.
[goword.log](https://github.com/pingcap/tidb/files/6820167/goword.log)

Is there any issue to track making `goword` happy?

**Comment by mmyj** [comment]
https://github.com/pingcap/tidb/issues/3120

File: executor/insert_common.go [hunk 2]
```diff
@@ -86,7 +86,7 @@ type InsertValues struct {
 
 	stats *InsertRuntimeStat
 
-	// LoadData use two goroutines. One for generate batch data,
+	// isLoadData LoadData use two goroutines. One for generate batch data,
```
Comment (mmyj) [inline executor/insert_common.go:89]: isLoadData indicates whatever current goroutine is use for generating batch data. LoadData use two goroutines...

**Review by mmyj — APPROVED** [review]
lgtm

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**